### DISCUSSION TITLE
Disable JSX transform

### DISCUSSION THREAD

**Discussion by JoviDeCroock** [swc-project/swc#1422] [Q&A]
Hey,

Thank you for the amazing work on swc, wanted to ask if it's possible to disable the transformation of JSX, this because we aim to transform JSX to [`htm`](https://github.com/developit/htm) calls rather than `h()` calls.

If we could disable the transform by not specifying `transform.React` that'd be amazing, this would let the parser deal with TSX/JSX files but not transform them.

**Comment by duncanbeevers** [comment]
@JoviDeCroock Is it sufficient to use the existing `pragma` and `pragmaFrag` options to specify your own transform instead of the default `React.createElement` and `React.Fragment` transforms?

https://swc.rs/docs/configuring-swc#jsctransformreact

**Reply by JoviDeCroock** [reply]
No, we transform to string literals like 

```
console.log(<h1 id="hello">Helloworld!</h1>);
// becomes
console.log( html`<h1 id=hello>Hello world!</h1>` );
```
